common/main-container/maincontainer-fr.html

```diff
@@ -0,0 +1,91 @@
+---
+{
+	"altLangPage": "main-container-fr.html",
+	"dateModified": "2023-07-04",
+	"description": "Documentation sur l'utilisation du conteneur principal.",
+	"language": "fr",
+	"title": "Example du conteneur principal"
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<h2>Conteneur Principal</h2>
+<main property="mainContentPage">
+
+    <!-- heading level 1 -->
+  
+    <!-- content page -->
+  
+    <!-- Page details section [Must be configured as a <section> element in this situation -->
+    <p>Cet espace désigné sert de conteneur HTML principal, vous offrant la possibilité d'incorporer de manière transparente tout le contenu pertinent pour cette page.</p>
+  
+</main>
+
+<h2>Conteneur Principal avec "article"</h2>
+<main property="mainContentPage">
+    <article>
+      <!-- heading level 1 -->
+  
+      <!-- content page -->
+  
+      <!-- Page details section [Must be configured as a <footer> element in this situation] -->
+    
+        <h3>Article</h3>
+        <p>Cela représente un article inclus à l'intérieur du conteneur principal.</p>
+  
+    </article>
+</main>
+
+<h2>Conteneur Principal avec "section"</h2>
+<main property="mainContentPage">
+    <section>
+      <!-- heading level 1 -->
+  
+      <!-- content page -->
```

Comment:
When you will move this example in its own page, please add the h1 and for the content you can add a paragraph of [lorem ipsum](https://www.lipsum.com/)